test_files/promiseconstructor/promiseconstructor.ts

```diff
@@ -0,0 +1,3 @@
+function f(promiseCtor?: typeof Promise): Promise<void> {
```

Comment:
Can you add a comment to this file about what the test is concerned with?  It's just trying to use the PromiseCtor type?

Reply:
Done.